lib/vault.ts

```diff
@@ -135,15 +139,48 @@ export async function listAllSubmissions(
         lastEvaluatedKey = response.LastEvaluatedKey;
       }
     }
+
     logEvent(
       ability.userID,
       { type: "Response" },
       "ListResponses",
       `List all responses for form ${formID}`
     );
-    return accumulatedResponses;
+
+    const numOfUnprocessedSubmissions = accumulatedResponses.filter((submission) =>
+      [VaultStatus.NEW, VaultStatus.DOWNLOADED].includes(submission.status)
```

Comment:
VaultStatus.PROBLEM should also be included in this list.  All problem responses should be resolved before deleting the form as well.